cli/dcos-spark/main.go

```diff
@@ -43,12 +43,7 @@ type SparkCommand struct {
 }
 
 func (cmd *SparkCommand) runSubmit(a *kingpin.Application, e *kingpin.ParseElement, c *kingpin.ParseContext) error {
-	marathonConfig, err := fetchMarathonConfig()
```

Comment:
FWIW it may be necessary to make dispatcher additions in order for this to be removable? I don't remember for sure.

Effectively the dispatcher could grab these values from its environment and include them in the task automatically, instead of requiring the CLI to pass them in from the outside.